js/login.js

```diff
@@ -0,0 +1,123 @@
+const MASTER_ACCOUNT = {
+  EMAIL: 'test@codeit.com',
+  PASSWORD: 'codeit101'
+}
+
+const REGEX = {
+  EMAIL: /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/i,
+  PASSWORD: '',
+}
+
+const ERROR_MSG = {
+  BLANK: {
+    EMAIL: '이메일을 입력해 주세요.',
+    PASSWORD: '비밀번호를 입력해 주세요.'
+  },
+  INVALID: {
+    EMAIL: '올바른 이메일 주소가 아닙니다.'
+  },
+  INCORRECT:{
+    EMAIL: '이메일을 확인해 주세요.',
+    PASSWORD: '비밀번호를 확인해 주세요.'
+  }
+}
```

Comment:
상수화 좋네요! 
이런 코드들이 자바스크립트를 모듈로 다루면 src/constants 경로 아래에 들어간다고 생각해주시면 될 것 같아요~

아래는 사소한 부분인데 생각해보니 메세지는 MSG보다 MESSAGE가 더 괜찮은 것 같기두 하네요!
물론 취향이라고 생각해요!

그리고 에러의 종류별이 아닌, 어디서 발생하는 에러인지에 대한 기준으로 객체의 key값이 EMAIL, PASSWORD가 되는 것도 괜찮을 것 같아요!
```suggestion
const ERROR_MESSAGE = {
  BLANK: {
    EMAIL: '이메일을 입력해 주세요.',
    PASSWORD: '비밀번호를 입력해 주세요.'
  },
  INVALID: {
    EMAIL: '올바른 이메일 주소가 아닙니다.'
  },
  INCORRECT:{
    EMAIL: '이메일을 확인해 주세요.',
    PASSWORD: '비밀번호를 확인해 주세요.'
  }
}
```

Reply:
selector로 사용되는 문자열도 상수화를 하기도 해요~